docs/integrations/destinations/s3.md

```diff
@@ -87,7 +87,7 @@ At this time, object-level permissions alone are not sufficient to successfully
 <!-- env:cloud -->
 
 :::note
-This authentication method is currently in the testing phase. To enable it for your workspace, please contact our Support Team.
+Currently, this feature is available only for the users in a Sales Assist workflow. Please [contact the Sales team](https://airbyte.com/company/talk-to-sales) if you are interested in using this.
```

Comment:
```suggestion
S3 authentication using an IAM role member must be enabled by a member of the Airbyte team. If you'd like to use this feature, please [contact the Sales team](https://airbyte.com/company/talk-to-sales) for more information.
```

Reply:
I think "users in the Sales Assist workflow" reads like a sales jargon phrase. Rewording to avoid that phrase would be more user-centric.

Reply:
this sounds way better. I think we still need @nataliekwong to approve